sdcm/cluster.py

```diff
@@ -5426,7 +5426,7 @@ def install_scylla_monitoring_prereqs(self, node):  # pylint: disable=invalid-na
         elif node.distro.is_debian:
             node.install_package(
                 package_name='apt-transport-https ca-certificates curl software-properties-common gnupg2')
-            node.remoter.sudo('curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -', retry=3)
+            node.remoter.sudo('curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -', retry=3)
```

Comment:
In such a case, IMHO, better to wrap the whole command in `bash -c "..."`.
Doing so we will not need to keep in mind that we use piping here with it's own "sudo". Which is error-prone.

Reply:
Fixed and retested here https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/debian11-sanity-test/217